### ISSUE TITLE
Checksum failure with norm_1.0-9.5 while building R-4.1.2-foss-2021b.eb

### ISSUE THREAD

**Issue by rmarable** [easybuilders/easybuild-easyconfigs#15124]
R-4.1.2-foss-2021b.eb is failing to build because of a checksum verification failure with norm_1.0-9.5

Here is the exact error:

== 2022-03-12 00:02:44,538 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): Checksum verification for extension source norm_1.0-9.5.tar.gz failed (at easybuild/framework/easyblock.py:631 in collect_exts_file_info)

The issue is resolved by updating the SHA256 checksum from:

```
    ('norm', '1.0-9.5', {
        'checksums': ['305cbf007f3905cfd535ed9bf5ae3e2995e228cc8883d6482e5d3a2f02814106'],
    }),
```
to:
```
    ('norm', '1.0-9.5', {
        'checksums': ['45016701291a5851d42452fc3ae682ff9bf86b0a9ba4619dc8b36a0187e4d0e4'],
    }),
```

Thanks,

Rodney

**Comment by rmarable** [comment]
I see this has been addressed already with the addition of the correct checksum.

**Comment by boegel** [comment]
indeed, fixed in #14987
